pizero/runner.py

```diff
@@ -0,0 +1,23 @@
+import io
+import time
+import picamera
+from PIL import Image
+import requests
+
+
+url = "http://192.168.1.62:8000/file" # api url
```

Comment:
Considering, the change in docker compose, should not it be 80 ?

Reply:
So far, okay for the url as it is, but will be in the end in a config file or var env ! (I might came with a suggestion soon, a "fleet-config-file" to refer to (not versionned in the repo), and the url would based on those info)

Reply:
Also, does it work as it is ? 

as long as in `server/src/app/main.py` , l19, the inference route is prefixed as inference ` prefix="/inference`, should not it be
url = "http://192.168.1.62:8000/inference/file" ?

Reply:
Yes I need to a something to proper setup ip adress. About the port yes this is an error ;)

Reply:
Okay :)
Looks good to me then 👍